src/containers/Header/index.js

```diff
@@ -92,19 +39,101 @@ class Header extends React.Component {
    * But in the future we'll unify the header styles and have a single color. So it doesn't make
    * much sense to invest a lot of time improving this.
    */
-  invertColors() {
+  _invertColors() {
     return ['/', '/about'].includes(this.props.location.pathname);
   }
 }
+Header = withRouter(Header);
+export default Header;
 
-Header = connect(
-  ({ download: { dataSet, isLoading } }) => ({
-    totalSamples: getTotalSamplesAdded({ dataSet }),
-    isLoading
+let HeaderLinks = ({ itemClicked, totalSamples, fetchDataSet, location }) => {
+  return (
+    <ul className="header__menu">
+      <HeaderLink
+        to="/"
+        onClick={itemClicked}
+        location={location}
+        activePath={['/results']}
+      >
+        Search
+      </HeaderLink>
+      <HeaderLink to="/api" onClick={itemClicked} location={location}>
+        API
+      </HeaderLink>
+      <HeaderLink to="/docs" onClick={itemClicked} location={location}>
+        Docs
+      </HeaderLink>
+      <HeaderLink to="/about" onClick={itemClicked} location={location}>
+        About
+      </HeaderLink>
+      <li className="header__link">
+        <Link
+          className="button button--secondary header__link-button"
+          to="/download"
+          onClick={itemClicked}
+        >
+          Download Dataset
+          <Loader fetch={fetchDataSet}>
+            {({ isLoading }) =>
+              !isLoading && (
+                <div className="header__dataset-count">{totalSamples}</div>
+              )
+            }
+          </Loader>
+        </Link>
+      </li>
+    </ul>
+  );
+};
+HeaderLinks = connect(
+  ({ download: { dataSet } }) => ({
+    totalSamples: getTotalSamplesAdded({ dataSet })
   }),
   {
     fetchDataSet
   }
-)(Header);
-Header = withRouter(Header);
-export default Header;
+)(HeaderLinks);
+
+let HeaderLink = ({ to, onClick, children, location, activePath = [] }) => {
+  return (
+    <li
+      className={classnames('header__link', {
+        'header__link--active':
+          location &&
+          (location.pathname === to ||
+            activePath.some(x => location.pathname === x))
```

Comment:
I wonder whether
```
activePath.includes(location.pathname)
```
will be more readable.

Reply:
way better, I'll update it, thanks!